src/components/available-organisation-units-tree/AvailableOrganisationUnitsTree.js

```diff
@@ -65,23 +79,34 @@ class AvailableOrganisationUnitsTree extends PureComponent {
     }
 
     render() {
-        if (this.state.rootWithMembers) {
+        if (!this.state.rootsWithMembers) {
+            return <span>{i18n.t('Updating Organisation Units Tree...')}</span>
+        }
+
+        if (this.state.rootsWithMembers.length < 1) {
             return (
-                <div className={styles.tree}>
+                <p>
+                    {i18n.t(
+                        'You do not have access to any organisation units.'
+                    )}
+                </p>
+            )
+        }
+
+        return (
+            <div className={styles.tree}>
+                {this.state.rootsWithMembers.map(rootOrgUnit => (
                     <OrgUnitTree
+                        key={rootOrgUnit.id}
                         hideMemberCount={Boolean(true)}
-                        root={this.state.rootWithMembers}
+                        root={rootOrgUnit}
                         selected={this.state.selected}
-                        initiallyExpanded={[
-                            `/${this.state.rootWithMembers.id}`,
-                        ]}
+                        initiallyExpanded={[`/${rootOrgUnit.id}`]}
                         onSelectClick={this.handleOrgUnitClick}
                     />
-                </div>
-            )
-        }
-
-        return <span>{i18n.t('Updating Organisation Units Tree...')}</span>
+                ))}
```

Comment:
I think you could simplify this part. Currently you are mapping over the roots and displaying individual `OrgUnitTree` components. 

However, I believe there is also the `OrgUnitTreeMultipleRoots` which does this for you already. That'd be slightly easier.

Reply:
Yeah wasn't aware of that actually, but it does basically the exact same thing when looking at the code (except with some validation for props). I will try to use that.

Reply:
Actually I will keep it as is, as it doesn't seem possible to initially expand all roots with that component.